proxy/README.md

```diff
@@ -0,0 +1,37 @@
+# OPEA Pipeline Proxy
+
+OPEA Pipeline Proxy is a replacement of the default Istio proxy with additional features designed specifically for OPEA RAG pipelines.
```

Comment:
I suggest "an enhancement" since it's still an envoy proxy essentially.

Reply:
Updated, thanks.